languages/scala/recursive_descent/Parser_scala_recursive_descent.ml

```diff
@@ -2730,6 +2743,14 @@ and parseTry in_ : stmt =
     | Kcatch ii -> (
         nextToken in_;
         match in_.token with
+        | LBRACE _ when isEllipsisCase in_ ->
```

Comment:
please add a comment like (* semgrep: this is only needed to parse semgrep patterns! *)
actually you might want to guard it with Flag_parsing.sgrep_guard

Reply:
good catch!